x-pack/plugins/ml/common/types/field_types.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// import {
+//   ES_FIELD_TYPES,
+//   KBN_FIELD_TYPES,
+//   ML_JOB_FIELD_TYPES
+// } from '../constants/field_types';
+
+export const getFieldTypeByKeyword = () => {};
```

Comment:
this `types` directory is for common TypeScript type definitions, i don't think this file belongs in here.

Reply:
@jgowdyelastic, thank you for the remark! I'm moving such functionality to field_types_util.js (which already exists), -- does it sound like a good decision?